fluentd/configs.d/filter-pre-mux-client.conf

```diff
@@ -5,6 +5,8 @@
   shared_key "#{File.open('/etc/fluent/muxkeys/shared_key') do |f| f.readline end.rstrip}"
   secure yes
   ca_cert_path /etc/fluent/muxkeys/ca
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
```

Comment:
No, none of these conf files are in ansible currently.  The only fluentd conf files we have in ansible are the main fluent.conf and the secure_forward.conf for sending logs outside of the cluster.